pkg/synopsysctl/cmd_provision.go

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2019 Synopsys, Inc.
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package synopsysctl
+
+import (
+	"fmt"
+	"github.com/blackducksoftware/synopsys-operator/pkg/polaris"
+	"github.com/blackducksoftware/synopsys-operator/pkg/util"
+	"github.com/spf13/cobra"
+	v12 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type polarisProvisionJob struct {
+	coverityLicensePath                          string
+	organizationProvisionOrganizationDescription string
+	organizationProvisionOrganizationName        string
+	organizationProvisionAdminName               string
+	organizationProvisionAdminUsername           string
+	organizationProvisionAdminEmail              string
+}
+
+var ctlProvisionJob polarisProvisionJob
+
+var provisionCmd = &cobra.Command{
```

Comment:
Are we sure that we want `./synopsysctl provision polaris` and not `./synopsysctl update polaris add-organization`? I'm just thinking this because we recently said that we want to be more careful about how we structure commands